lib.sh

```diff
@@ -105,6 +106,8 @@ NCUSER=ncadmin
 UNIXUSER=$SUDO_USER
 UNIXUSER_PROFILE="/home/$UNIXUSER/.bash_profile"
 ROOT_PROFILE="/root/.bash_profile"
+# Name in trusted_config
+NCDOMAIN=$(nextcloud_occ_no_check config:system:get overwrite.cli.url | sed 's|https://||;s|/||')
```

Comment:
are you sure it is a good idea to get this value every time we get the library?

Reply:
We might use it in more places, and I think it was a good variable.

Reply:
maybe wrap it in a function then?

Reply:
Sure, go ahead!

Reply:
I'll merge this now, we can fix details later.